### PR TITLE
:sparkles: Add full ICU support to CircleCI 2.0 Node 9

### LINKED ISSUE #12: Add custom image with full-icu for CircleCI 2.0

### PR DESCRIPTION
**Pull request by charlyx**
* added NPM package full-icu
* set NODE_ICU_DATA to global installation of full-icu

fix #12

### CODE REVIEW

**Comment by jlandure** [comment]
What's the benefits of using the `circleci:node` docker image ?

**Review by jlandure — APPROVED** [review]

File: circleci/hooks/post_push [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+tagStart=$(expr index "$IMAGE_NAME" :)
+repoName=${IMAGE_NAME:0:tagStart-1}
+
+for tag in {full-icu,icu,i18n}; do
```
Comment (jlandure) [inline circleci/hooks/post_push:7]: les tags vont être en conflit avec les tags de mon répertoire "full-icu"...

Comment (jlandure) [reply circleci/hooks/post_push:7]: Peut-être la renommer en "circleci-". Qu'en pensez-vous @charlyx @ebriand ?

**Review by jlandure — DISMISSED** [review]

**Comment by charlyx** [comment]
I’m using circleci:node so I make sure my image meets all the requirements for CircleCI to work as expected. Also the aim of this image is to bring full-icu support when using node in CircleCI 2.0.

**Review by jlandure — APPROVED** [review]

**Pull request merged**